cron/src/DailyCron.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
These files do not respect the project code style convention, please fix this and all the other Cron classes.
https://github.com/supertuxkart/stk-addons#php

Reply:
Sorry, I will fix it.
May I open new pull request for checking code style in CI?

Reply:
You mean a new PR to fix this PR code style?

Reply:
To add style checker to CI pipeline. You know phpcs?

Reply:
I don't know about it but it sounds good, please fix your issues on this PR first though :).

Reply:
Coding style fixed in cron classes.
PR for phpcs added.